features/lividDetector.js

```diff
@@ -0,0 +1,26 @@
+import { drawBox } from "./glowingEffect";
+
+class LividDetector {
+    constructor() {
+        this.name = "Livid Detector";
+        this.triggers = {"renderWorld": [{"func": this.renderWorld, "criteria": null}]};
+    }
+    
+    renderWorld(partialTicks) {
```

Comment:
this should be not detecting livids every frame, change it to every tick instead